packages/micropip/test_micropip.py

```diff
@@ -228,3 +228,18 @@ def test_install_different_version2(selenium_standalone_micropip):
         `);
         """
     )
+
+
+@pytest.mark.parametrize("jinja2", ["jinja2", "Jinja2"])
+def test_install_mixed_case2(selenium_standalone_micropip, jinja2):
```

Comment:
Maybe let's only test the capitalized version (`PyTz`) to make sure it's clearly not the right capitalization, as `selenium_standalone` tests are very slow so we should avoid adding that for cases that are already covered (e.g. `pytz` above)

Reply:
It's important that the package name has a capital letter in it though. We can't make requests to `build/jinja2.js` because that isn't a real file, the file is called `build/Jinja2.js`.